changesLog.js

```diff
@@ -1,34 +1,75 @@
+const octokit = require('./githubAPI/octokit');
 const commander = require('commander');
-const { getIssuesByMilestone } = require('./changesLogGenerator');
+const fs = require('fs');
+const { fetchIssuesInMilestone, generateChangesLog } = require('./helpers');
 
-function commaSeparatedList(value, dummyPrevious) {
+function commaSeparatedList(value) {
   return value.split(',');
 }
```

Comment:
Any reason for ES5 function synxtax? IMO would be nicer to keep consistent with other repos and use arrow syntax?

```suggestion
const commaSeparatedList = value => value.split(',');
```
```

Reply:
I agree. I was only avoiding installing packages to convert things from ES6, but that is accepted for ES5.